client/src/app/auth/firebase-auth.service.ts

```diff
@@ -33,35 +33,28 @@ export class FirebaseAuthService extends AuthService {
     return this.requireAuth().pipe(take(1));
   }
 
-  refreshToken() {
-    return fromPromise(<Promise<any>>firebase.auth().currentUser.getIdToken(true));
-  }
```

Comment:
While we might not need this API with the new auth protocol anymore as of this PR, there's no need to remove it at this point.

If we decide to remove later, we can do it in a separate PR.